### DISCUSSION TITLE
I need help for make minimal size .exe using PyInstaller

### DISCUSSION THREAD

**Discussion by Avocadocs** [pyinstaller/pyinstaller#6756] [Help]
Hello,
I'm trying to make mininmal size .exe using PyInstaller, for achieve this I'm using venv. I installed only required packages. Then I use `pyinstaller --clean -F knifegen.spec`. Here is .spec
[knifegen.spec.txt](https://github.com/pyinstaller/pyinstaller/files/8504779/knifegen.spec.txt)
 
 Problem is I'm getting ~60 MB .exe file which is too big. I retryed but always same problem, I'm sure I'm using pyinstaller version from venv which is activated. So maybe problem is in my imports ?
```
import tkinter as tk
from tkinter import ttk, filedialog, messagebox
import subprocess
import cv2
from ctypes import windll
from PIL import Image
import os
import requests
```
Thanks for your help!

**Comment by bwoodsend** [comment]
OpenCV is huge and it depends on NumPy which is also huge so 60MB sounds like a pretty good deal to me.  If you're on Windows the you can install NumPy vanilla (built without openblas) from [here](https://www.lfd.uci.edu/~gohlke/pythonlibs/#numpy). If you're on Linux, you can probably run `strip` on most of the `.so` files.
